### PR TITLE
docs(Dropdown): fix world icon in search example

### PR DESCRIPTION
**Pull request by aabustamante**
the example have a note where it says that needs a world icon

### CODE REVIEW

**Comment by aabustamante** [comment]
I renamed "languageOptions" to "languages" because the row had more than 120 characters and tests will fail

File: docs/app/Examples/modules/Dropdown/common.js [hunk 1]
```diff
@@ -337,7 +337,7 @@ export const countryOptions = [
   { key: 'zw', value: 'zw', flag: 'zw', text: 'Zimbabwe' },
 ]
 
-export const languageOptions = [
+export const languages = [
```
Comment (levithomason) [inline docs/app/Examples/modules/Dropdown/common.js]: For consistency with all the other exports in this file, let's restore this const name.

**Review by levithomason — APPROVED** [review]
Just one comment regarding naming conventions.

File: docs/app/Examples/modules/Dropdown/Types/DropdownExampleSearchDropdown.js [hunk 2]
```diff
@@ -4,15 +4,9 @@ import { Dropdown } from 'semantic-ui-react'
...
- * NOTE: In this example, the dropdown should contain a label on the
- * left of the text:
- * <i class="world icon"></i>
- *
- * @returns {Element}
- */
 const DropdownExampleSearchDropdown = () => (
-  <Dropdown text='Select Language' search floating labeled button className='icon' options={languageOptions} />
+  <Dropdown text='Select Language' 
+  search floating labeled button className='icon' icon='world' options={languageOptions} />
```
Comment (levithomason) [inline docs/app/Examples/modules/Dropdown/Types/DropdownExampleSearchDropdown.js]: If props need to be on separate lines, one prop per line please.  The closing tag should also be on its own line.

```jsx
<Dropdown
  text='Select Language'
  search
  floating
  labeled
  button
  className='icon'
  icon='world'
  options={languageOptions}
/>
```

Comment (aabustamante) [reply docs/app/Examples/modules/Dropdown/Types/DropdownExampleSearchDropdown.js]: ok, thanks. I will change it.

Comment (aabustamante) [reply docs/app/Examples/modules/Dropdown/Types/DropdownExampleSearchDropdown.js]: @levithomason  thanks for helping me. I already made the changes you suggested me and now test passes successfully

**Comment by levithomason** [comment]
Thanks for the update.

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.68.4`.

**Pull request merged**